pkg/manager/internal/controllers/organizationroletemplate_controller.go

```diff
@@ -62,6 +98,10 @@ func (r *OrganizationRoleTemplateReconciler) Reconcile(req ctrl.Request) (ctrl.R
 			Reason:  "SetupComplete",
 			Message: "OrganizationRoleTemplate setup is complete.",
 		})
+		isChanged = true
+	}
+
+	if isChanged {
```

Comment:
If you insist on having variable has the status changed, and update the status on it, call it by more established name --> dirty. (e.g. dirty pages from OS class, https://en.wikipedia.org/wiki/Page_cache ). Thus the most appropriate name I could think of is `dirty`. (or `dirtyStatus` or some variation thereof). What do you think?

Reply:
I prefer `isChanged`, I think it is easy to understand, `dirty` is also ok, but I don't really see this is used in projects, I always see `isChanged` or something like that

Reply:
where have you seen `isChanged` used? It's fine as well, it's descriptive in what it does.

Reply:
I have seen kubernetes uses `changed`, if you think that is better, we can also use that.

Reply:
sure, since kubernetes uses `changed` we should too.

Reply:
done